### ISSUE TITLE
Github issue | unable to participate 

### ISSUE THREAD

**Issue by atomGit** [privacyguides/privacyguides.org#75]
i cannot participate in the following discussions without registering my email address...

Suggest a New Provider or Software
Ask a Question
Share an Idea

...and i'm not going to register my mail address - microsoft is a f'n evil bastard of a company and allot of developers (including myself) have left this place since they acquired it

if you'll allow me to post comments regarding the above topics as issues instead of discussions, that would be ok, however, given that Privacy Guides is about privacy, and microsoft is absolutely not concerned with user privacy -- quite the opposite -- i humbly ask that you consider moving to a more ethical, privacy respecting platform, such as [Codeberg](https://codeberg.org/)

that said, if you care to read what i initially wanted to post....

**VPN**

[Providers / VPN](https://www.privacyguides.org/providers/vpn/) - i would recommend having a good look at [AzireVPN](https://www.azirevpn.com/manager/auth/register) for the following reasons...

* their focus is on WireGuard
* they own their own servers rather than rent them and they install themselves
* they physically seal hardware ports to prevent tampering
* the entire system is run in RAM - hard drives are removed
* no blocked ports
* they run SOCKS5 proxy services on all their servers
* DNS is provided through the tunnel

they're a smaller company and so not allot of servers, but my service has been extremely good and stable thus far

device setup on some hardware, like routers, can be problematic but i assume their apps work ok

they aren't always quick to reply to support requests

regarding the opening text on the VPN page...

> If you're looking for added security, you should always ensure you're connecting to websites using encrypted DNS and HTTPS. A VPN is not a replacement for good security practices.

perhaps that should be modified to include that some/many VPN providers do offer secure DNS

**Comment by jonaharagon** [comment]
@atomGit are you able to reply to open discussions?

**Comment by atomGit** [comment]
negitive

**Comment by Loki-L1130** [comment]
> 

> Negative

Then, Why not use Free and Open Source, Email Alias Providers such as SimpleLogin and AnnonAddy ?

**Comment by atomGit** [comment]
thanks for the suggestions

still, i find it odd that a privacy focused platform would be using the services of M$

i understand that github offers more features and flexibility than most other platforms, however that's exactly how people are subdued into using services and devices that are anti-privacy - for those that care about privacy, sometimes sacrifices need to made

**Comment by SrEstegosaurio** [comment]
I compleatly agree.
I still own this account just because some things like this in wich I can only participate using GitHub. 

But for my point of view: It does **NOT** feel logical to have a privacy focused site on GitHub, and it gets even worse when already exist privacy focused platforms for code hosting & that (like: Codeberg, Gitea, Sourcehut, Notabug, GOGS...) 

I hope they move from this hell...

**Comment by SrEstegosaurio** [comment]
> > Negative
> 
> Then, Why not use Free and Open Source, Email Alias Providers such as SimpleLogin and AnnonAddy ?

It stills not a great solution. I mean, if is **completely necessary** well... But using services like AnnonAddy, SimpleLogin etc etc _without selfhosting them_ is having to put trust on another point, witch is far from ideal... 

I think that the solution is migrating the project to whatever privacy respecting platform.

**Comment by Mikaela** [comment]
The traditional argument for using GitHub is that it's necessary for visibility/discoverability of the project and I hear often that one has to have a GitHub account for getting employed in ICT.

In [my way](https://github.com/Mikaela) I think the solution is using Gitea [which added push mirrors in 1.15.0](https://blog.gitea.io/2021/08/gitea-1.15.0-is-released/) in an [Indieweb POSSE style](https://indieweb.org/POSSE) (Post On your own Site, Syndicate Elsewhere) so that code and issues live in e.g. git.privacyguides.org (I remember PrivacyTools had one, but not why exactly it was shut down) while it automatically pushes changes to GitHub e.g. every 8 hours.

Should someone regardless send PRs at GitHub, they can be merged through git command-line by adding a remote or adding a `.patch` to the end of PR address especially if it's a small change without requiring an Gitea account. Then again Gitea can be configured to allow GitHub login.

**Comment by Loki-L1130** [comment]
> It stills not a great solution. I mean, if is **completely necessary** well... But using services like AnnonAddy, SimpleLogin etc etc _**without selfhosting them**_ is having to put trust on another point, witch is far from ideal...

SimpleLogin published the Source Code for their Server and you can Selfhost yourself. SimpleLogin is Open Source from Top to bottom. So, You can trust them, Blindly

**Comment by SrEstegosaurio** [comment]
> 
> > It stills not a great solution. I mean, if is **completely necessary** well... But using services like AnnonAddy, SimpleLogin etc etc _**without selfhosting them**_ is having to put trust on another point, witch is far from ideal...
> 
> SimpleLogin published the Source Code for their Server and you can Selfhost yourself. SimpleLogin is Open Source from Top to bottom. So, You can trust them, Blindly
> 
> 
> 

They are inmho really trustworthy, I mean, even their homepage is OpenSource. 
What I'm saying is that they still susceptible to laws and that stuff. And I think that being as _"minimalistic"_ as possible on the internet helps a lot protecting your privacy. 

I think that for some threat models it will be no compleatly Okey. But I agree that you can always selfhost it and then you don't have to put any kind of trust on anyone

**Comment by SrEstegosaurio** [comment]
> The traditional argument for using GitHub is that it's necessary for visibility/discoverability of the project and I hear often that one has to have a GitHub account for getting employed in ICT.
> 
> In [my way](https://github.com/Mikaela) I think the solution is using Gitea [which added push mirrors in 1.15.0](https://blog.gitea.io/2021/08/gitea-1.15.0-is-released/) in an [Indieweb POSSE style](https://indieweb.org/POSSE) (Post On your own Site, Syndicate Elsewhere) so that code and issues live in e.g. git.privacyguides.org (I remember PrivacyTools had one, but not why exactly it was shut down) while it automatically pushes changes to GitHub e.g. every 8 hours.
> 
> Should someone regardless send PRs at GitHub, they can be merged through git command-line by adding a remote or adding a `.patch` to the end of PR address especially if it's a small change without requiring an Gitea account. Then again Gitea can be configured to allow GitHub login.

I don't think that I fully understood what you said, sorry. But I think that I got the main idea _(or something like that)_: Are you referring to have a repo on Github and another in Gitea and have them _"synched"_ in some way? 

Because I can understand the issue about visibility and work _(and etc)_ but I think that having liberty to choose from where contribute would be cool.

**Comment by Mikaela** [comment]
> Are you referring to have a repo on Github and another in Gitea and have them "synched" in some way?

Basically my suggestion is to have a repository on (self-hosted) Gitea and have it send changes to read-only GitHub.

* ![Gitea push mirror example from Gitea blog](https://blog.gitea.io/demos/15157/push_mirror.png)
  * Gitea push mirror example from Gitea blog

> ...I think that having liberty to choose from where contribute would be cool.

I fear actively accepting contributions on multiple different places is not going to work, unless PrivacyGuides delegates a team member to solely copy issues around as that takes a lot of effort and time.

>> Should someone regardless send PRs at GitHub, they can be merged through git command-line by adding a remote or adding a .patch to the end of PR address especially if it's a small change without requiring an Gitea account. Then again Gitea can be configured to allow GitHub login.

By this I meant purely code, [for example PR 91 patch](https://github.com/privacyguides/privacyguides.org/pull/91.patch) or `git config --add remote.origin.fetch '+refs/pull/*/head:refs/remotes/origin/pr/*'` which are trivial to merge as long as the change isn't too big for the team to review without GitHub features.

**Comment by tauin** [comment]
There is the issue of participation, many people already have a github account, and having to sign up for another account inherently increases the barrier to entry of e.g submitting patches and issues, if PG wants to migrate to gittea, then i suggest that you keep issues open here so people can submit them, and then use a bot to submit them to the gittea instance with a bot (openmw did something similar, although it was static, i.e one and done)

**Comment by criadoperez** [comment]
I also vote on moving out of Github.
Serious privacy projects (like F-droid or Calyx) aren't here, because that would be inconsistent with what they are trying to accomplish. 
Repository can be mirrored to Github (and others) for visibility, but main development shouldn't be here.

It's like joining a privacy conference on Teams or Zoom... If I see that, I can't take them seriously...

What about Gitlab? Gitlab is open source. Github isn't.

**Comment by rusty-snake** [comment]
> What about Gitlab?

- bullies tor users
- uses ReCaptcha IIRC
- denies login if you use RFP
- disables issue search if you aren't logged in
- can not be viewed without javascript
